README.md

```diff
@@ -1,2 +1,33 @@
-# mobile-template
+# Mobile-Template
 In this proyect I will be creating the frist part  for the mobile version of my personal portfolio 
+
+## Built With
+- HTML
+- CSS
+
```

Comment:
- [Optional] Consider adding a getting started section to your readme file to provide the steps required to get a local copy of your project. This will make your code look more professional.